frontend/app_flowy/packages/appflowy_editor/example/lib/main.dart

```diff
@@ -123,6 +124,7 @@ class _MyHomePageState extends State<MyHomePage> {
                 'text/code_block': CodeBlockNodeWidgetBuilder(),
                 'tex': TeXBlockNodeWidgetBuidler(),
                 'horizontal_rule': HorizontalRuleWidgetBuilder(),
+                'network_image': NetworkImageNodeWidgetBuilder(),
```

Comment:
I think it's not necessary to add this code. We're using `ImageNodeBuilder` to render the image node now.

Reply:
Addressed in [3963bd3](https://github.com/AppFlowy-IO/AppFlowy/pull/1277/commits/3963bd3e99bbac2460c31c970ecc8acd88acc486)